modules/spock/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
```

Comment:
This is the new plugins DSL, we can of course use also the old one to keep styles more in sync?

Reply:
we use the new style as much as possible, I would keep it :)